### ISSUE TITLE
Period in Etcd Keynames

### ISSUE THREAD

**Issue by stongo** [containous/traefik#295]
When using Romulus as Kubernetes ingress controller, if internal dns addon is activated it produces keys like this in etcd for traefik

`/traefik/frontends/kube-system.kubernetes-dashboard.80`

Judging from log output, it would appear that everything after the first `.` is dropped by Traefik

`Configuration received from provider etcd: {\"backends\":{\"kube-system\":{}},\"frontends\":{\"kube-system\":{}}}"`

Which ultimately makes traefik not work.

Is this something that can be fixed in Traefik or should I look Romulus to substitute periods?

**Comment by stongo** [comment]
I don't think this is fixed yet. Still get errors using `containous/traefik:latest`

```
time="2016-04-18T19:06:21Z" level=warning msg="Error getting key /traefik/frontends/kube-system.kubernetes-dashboard.80/entrypoints Key not found in store, setting default empty"
time="2016-04-18T19:06:21Z" level=warning msg="Error getting key /traefik/frontends/kube-system.kubernetes-dashboard.80/passHostHeader Key not found in store, setting default false"
time="2016-04-18T19:06:21Z" level=debug msg="Configuration received from provider etcd: {\"backends\":{\"kube-system\":{}},\"kube-system\":{}}}"
time="2016-04-18T19:06:21Z" level=debug msg="Last etcd config received less than 2s, waiting..."
```

**Comment by stongo** [comment]
also I get a whole ton of those key Errors and traefik seems to hammer etcd

**Comment by emilevauge** [comment]
@stongo you right :'(
There was a bug in our `github.com/BurntSushi/toml` version.
This will be fixed by #186
